styles/global.css

```diff
@@ -3,30 +3,30 @@
 
 /* Global styles */
 :root {
-  --primary: #2f6ed3;
-  --primary-light: #f1f7ff;
-  --secondary: #27374c;
-  --secondary-light: #5842c4;
+  --primary: #2f6ed3 !important;
```

Comment:
you should totally avoid !important, why are using this in root?

Reply:
No i had a css error because of something else so I was testing stuff out, then forgot to remove these